config/sample.env

```diff
@@ -79,3 +79,6 @@ WAREHOUSE_JOBS_DB_PORT=5432
 WAREHOUSE_JOBS_DB_DB_NAME=jobsdb
 
 WAREHOUSE_URL=http://localhost:8082
+
+SSL_CERTIFICATE_FILE_PATH=<certificate_file_path>
```

Comment:
Rename these to KAFKA_SSL_CERTIFICATE_FILE_PATH

Reply:
Incorporated.